lib/connectors/base/connector.rb

```diff
@@ -26,13 +26,14 @@ def self.service_type
         raise 'Not implemented for this connector'
       end
 
-      def initialize(local_configuration = {})
+      def initialize(local_configuration:, connector_settings:)
```

Comment:
I think I am having a hard time trying to understand what should be there in the connector_settings. Are those the settings that are persisted in elasticsearch? The configurable fields? Or something else? I think the naming could be a bit clearer.